incubator/jaeger/templates/common-cm.yaml

```diff
@@ -0,0 +1,83 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ template "jaeger.fullname" . }}
+  labels:
+    app: {{ template "jaeger.name" . }}
+    jaeger-infra: common-configmap
+    chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
+    heritage: {{ .Release.Service }}
+    release: {{ .Release.Name }}
+data:
+  cassandra.contact-points: {{ template "cassandra.contact_points" . }}
+  cassandra.datacenter.name: {{ .Values.cassandra.config.dc_name | quote }}
+  cassandra.keyspace: {{ printf "%s_%s" "jaeger_v1" .Values.cassandra.config.dc_name | quote }}
+  cassandra.schema.mode: {{ .Values.schema.mode | quote }}
+  cassandra.servers: {{ template "cassandra.fullname" . }}
```

Comment:
This forces the external or previously deployed cassandra host to be <relname>-cassandra. This should be set to  `.Values.storage.cassandra.host`.

Reply:
Seems like we might need another define in helpers like you did for elasticsearch to toggle between using the `template "cassandra.fullname"` and `.Values.storage.cassandra.host`.

Reply:
Good catch! I've tested provisioned cassanrad, external ES and provisioned ES. This was the only scenario that I haven't tested. I'll fix it right away. Thank you very much for catching this!